src/main/java/com/project/poorlex/dto/battle/BattleCreateRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.project.poorlex.dto.battle;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Builder
+public class BattleCreateRequest {
+
+    private String name;
+
+    private int budget;
+
+    private int total;
+
+    private String description;
+
+    private LocalDateTime endDate;
+
+}
```

Comment:
방 제목
```suggestion
@Getter
@Builder
public class BattleCreateRequest {

    @Size(min=2, max=12)
    private String name;

    private int budget;

    private int total;

    private String description;

    private LocalDateTime endDate;

}
```
name 필드와 같이 기능정의서 참고하여 Validation 추가해주시면 좋아 보입니다!